Rakefile

```diff
@@ -39,6 +39,10 @@ task :test do
   raise unless system "nose2"
 end
 
+task :unittest do
```

Comment:
Just because I find myself typing `rake unittest` half the time and `rake test` half the time....